packages/commonwealth/client/scripts/navigation/customDomainRoutes.tsx

```diff
@@ -424,8 +424,8 @@ const customDomainRoutes = () => {
       element={<Navigate to="/profile/id/:profileId" />}
     />,
     <Route
-      path="/:scope/profile/id/:profileId/edit"
-      element={<Navigate to="/profile/id/:profileId/edit" />}
+      path="/:scope/profile/edit"
+      element={<Navigate to="/profile/edit" />}
```

Comment:
these 3 routes has to be added/modified in customDomainRoutes as well

Reply:
The routes are in both custom and common DomainRoutes -- I don't see what needs to be done?

Reply:
thats my bad!